metric_learn/lmnn.py

```diff
@@ -203,6 +206,9 @@ def _find_impostors(self, furthest_neighbors):
         tmp = np.ravel_multi_index((i,j), shape)
         i,j = np.unravel_index(np.unique(tmp), shape)
       impostors.append(np.vstack((in_inds[j], out_inds[i])))
+    if len(impostors) == 0:
+        """No impostors detected"""
+        return []
```

Comment:
No need for a docstring here. Also, you can just return `impostors` instead of returning a new empty list.